view/src/components/produce.js

```diff
@@ -492,79 +513,97 @@ class produce extends Component {
-                                                View{' '}
-                                            </Button>
-                                            <Button size="small" color="primary" onClick={() => this.handleEditClickOpen({ produce })}>
-                                                Edit
-                                            </Button>
-                                            <Button size="small" color="primary" onClick={() => this.deleteTodoHandler({ produce })}>
-                                                Delete
-                                            </Button>
-                                        </CardActions>
-                                    </Card>
-                                </Grid>
-                            ))}
-                        </Grid>
+								<SearchResults
+									value={value}
+									data={data}
+									renderResults={results => (
+										<div>
+                            				{results.map((produce) => (
+												<Grid item xs={12}key={produce.produceId}>
+													<Card className={classes.root} variant="outlined">
+														<CardContent>
+															<Typography variant="h5" component="h2">
+																{produce.name}
+															</Typography>
+															<Box display="flex" flexDirection="row" flexWrap="wrap" p={0} m={0}>
+																<Box p={3}>
+																	<Typography className={classes.pos} color="textSecondary">
+																		Shipping Temperatures in Reefer (°F): 
+																	</Typography>
+																	<Typography variant="body2" component="p">
+																		Maintenance Temperature: {produce.shippingMaintenanceTemperatureLow} - {produce.shippingMaintenanceTemperatureHigh}
+																		<br />
+																		Preset Temperature: {produce.shippingPresetTemperature}
+																	</Typography>
+																</Box>
+																<Box p={3}>
+																	<Typography className={classes.pos} color="textSecondary">
+																		Pricing (in USD / lb):
+																	</Typography>
+																	<Typography variant="body2" component="p">
+																		USDA Price: ${produce.price}
+																		<br />
+																		Average Price Paid by Farmlink: ${produce.pricePaid}
+																	</Typography>
+																</Box>
+																<Box p={3}>
+																	<Typography className={classes.pos} color="textSecondary">
+																		Internal Statistics:
+																	</Typography>
+																	<Typography variant="body2" component="p">
+																		Amount Moved by FarmLink (lbs): {produce.amountMoved}
+																	</Typography>
+																</Box>
+															</Box> 
+															
+														</CardContent>
+														<CardActions>
+															<Button size="small" color="primary" onClick={() => this.handleViewOpen({ produce })}>
+																{' '}
+																View{' '}
```

Comment:
whats with the {' '}?

Reply:
Hmm so this part of the code came from @andrewhojel and is getting discussed in his PR by Brian at https://github.com/googleinterns/FarmLink/pull/14#discussion_r468924743

Reply:
A friendly reminder that we should not be reviewing code that's not written by you in a PR authored by you.  :)  That was one of the objectives of the version control guidelines I emailed you a while back.  I know it's a pain, but I think we need to be careful of things like this to prevent total chaos.

Reply:
Noted, thank you for reminding me of the version control guidelines and pointing this out. At first, I was confused to see that these lines were showing as my changes. I might be mistaken, but after looking at my commits, it seems to me that this appeared as my code because I indented these lines forward since I added a new component above these lines. Lines 510-564 of this diff/commit in red show me deleting and then “re-writing the indented version” in lines 541 to 603.

This seems to be the same situation as my last commit to this PR, when I ran prettier, where the indent changes make it look like the entire file is re-written by me. In addition, I just tried indenting lines in a separate branch and checking the diffs afterwards, and it seems to look the same as the red and green case of lines 510-564 and lines 541 to 603 here. 

If I add my search feature’s lines into Andrew's other files and have to indent these sections further again, should I include some kind of note to make it clear in my commit that I'm only indenting these lines? How would you all prefer for me to make it clear that I’m only indenting these lines (or would this be partially solved by having smaller chunks of code so the green and red sections would be easier to view?)

If this issue is more complicated than being able to resolve here, I can bring this up during standup tomorrow before I start adding the search/filtering to the other pages.

Reply:
I'm guessing if we wait for Andrew's PR to be merged, many of the indentation changes will magically resolved, since he'll be committing that as well. The real issue is that the point at which you branched didn't include his first run with prettier. It's possible you could rebase your branch to a different one of his commits to resolve it sooner, but it would likely be easier to wait til he merges.

This shouldn't really be an issue usually, this is just one of the pains of us all learning open source.

Reply:
Gotcha—thank you, that clears up my question about the prettier tabs. I believe that for these commented lines in particular though, I added an extra indentation in order to put Andrew's cards/code into my new <SearchResults> component. I'm thinking that I'd still need to make this change in the other files (regardless of whether the other files have been run through prettier), so I'm worried that this same situation might arise again...

Reply:
The more I think about it, the more I wonder if the terror of rebase might be lesser than the terror of not rebasing.  In theory, if you rebase your most recent commit off of the most recent commit in the branch of Andrew's that you branched off of, that should generally protect you from situations like this.  And if he hasn't reformatted the files yet, then the reformatting is your change and not his, and so it's okay for us to review in your PR.  :)